src/app/ubs/ubs-admin/components/ubs-admin-order-payment/ubs-admin-order-payment.component.ts

```diff
@@ -11,45 +11,120 @@ import { AddPaymentComponent } from '../add-payment/add-payment.component';
   styleUrls: ['./ubs-admin-order-payment.component.scss']
 })
 export class UbsAdminOrderPaymentComponent implements OnInit, OnChanges {
-  message: string;
-  pageOpen: boolean;
-  @Input() overpayment: number;
   @Input() orderInfo: IOrderInfo;
   @Input() actualPrice: number;
   @Input() totalPaid: number;
-  orderId: number;
-  paidAmount: number;
-  unPaidAmount: number;
-  paymentInfo: IPaymentInfo;
-  paymentsArray: IPaymentInfoDtos[];
+
+  public message: string;
+  public pageOpen: boolean;
+  public orderId: number;
+  public overpayment: number;
+  public paidAmount: number;
+  public unPaidAmount: number;
+  public paymentInfo: IPaymentInfo;
+  public paymentsArray: IPaymentInfoDto[];
+
+  constructor(private orderService: OrderService, private dialog: MatDialog) {}
 
   ngOnInit() {
     this.orderId = this.orderInfo.generalOrderInfo.id;
     this.paymentInfo = this.orderInfo.paymentTableInfoDto;
-    this.paymentsArray = this.orderInfo.paymentTableInfoDto.paymentInfoDtos;
+    this.overpayment = this.paymentInfo.overpayment;
+    this.paymentsArray = this.paymentInfo.paymentInfoDtos;
     this.paidAmount = this.paymentInfo.paidAmount;
     this.unPaidAmount = this.paymentInfo.unPaidAmount;
+    this.setDateInPaymentArray(this.paymentsArray);
   }
 
-  constructor(private orderService: OrderService, private dialog: MatDialog) {}
-
   ngOnChanges(changes: SimpleChanges) {
     if (changes.overpayment) {
       this.message = this.orderService.getOverpaymentMsg(this.overpayment);
       this.overpayment = Math.abs(this.overpayment);
     }
   }
 
-  openDetails() {
+  public formatDate(date: string): string {
+    return date.split('-').reverse().join('.');
+  }
+
+  public setDateInPaymentArray(paymentsArray: IPaymentInfoDto[]) {
+    for (const payment of paymentsArray) {
+      payment.settlementdate = this.formatDate(payment.settlementdate);
+    }
+  }
+
+  public openDetails(): void {
     this.pageOpen = !this.pageOpen;
   }
 
-  setOverpayment(overpayment: number): void {
+  public setOverpayment(overpayment: number): void {
     this.message = this.orderService.getOverpaymentMsg(overpayment);
     this.overpayment = Math.abs(overpayment);
   }
 
-  openPopup(viewMode: boolean, paymentIndex?: number) {
+  public addZeroBeforeSingleDateValue(date: Date): string {
```

Comment:
Replace this method:
public getStringDate(date: Date): string {
    const options = { year: 'numeric', month: '2-digit', day: '2-digit' };

    return date.toLocaleDateString('en-GB', options).split('/').reverse().join('-');
  }